Documentation/install/rhel/installing-workers.md

```diff
@@ -127,12 +127,14 @@ Note: These settings may not be all inclusive and will not represent relative no
 
 ### Set SELinux to Permissive mode
 
-It is required to run SELinux in Permissive mode. Running in Enforcing mode will block permissions for worker nodes.
+It is required to run SELinux in Permissive mode. Running in Enforcing mode will block permissions for worker nodes. The following command will switch to Permissive mode until reboot.
```

Comment:
is there a good reason to recommend this at all? vs permanent solution?

Reply:
If they just edit the configuration file, they would have to reboot before it works.